### DISCUSSION TITLE
[typescript-operation] use-suspense-query

### DISCUSSION THREAD

**Discussion by younes0x53** [dotansimha/graphql-code-generator#4228] [General] [answered]
Hi,
I'm wondering if there is a easy way to wrap a custom query handler like https://github.com/Fasosnql/use-suspense-query
Any idea ?
Thank you

**Comment by dotansimha** [comment, marked as answer]
Hi @younes200 ! 

I guess what you are looking for is a way to integrate `useSuspenseQuery` with `typescript-react-apollo` plugin, right? (and not `typescript-operations`).

You can easily create a plugin that creates that for you, and integrates with `typescript-react-apollo` ;) Let me know if it's something that is relevant, we can help you with that.
